pkg/tenantctl/controller.go

```diff
@@ -414,12 +418,12 @@ func (c *Controller) checkNamespaceExists(name string) (bool, error) {
 }
 
 func (c *Controller) createTenantNamespace(config clientConfig, tenant *api.Tenant) (string, error) {
-	c.logPrintln(tenant, "creating new tenant namespace")
+	klog.V(3).Infof(c.formatLog(tenant, "creating new tenant namespace"))
```

Comment:
From my perspective every operation which modifies something should be logged with "info". According to the new documentation "3" means "trace".

But I wanted to test the log verbosity in real life anyway. Maybe the other log entries when a new tenant is created are enough.

Reply:
I will change this to the previous levels. Sorry for the confusion...
Anyway maybe we can test this together afterwards, due I never tested the coding life.

Reply:
That's what I also wanted to do - see the logging live. There you can see best if it is still too much logging, or if something ist missing.